.github/workflows/release_patch.yml

```diff
@@ -33,14 +33,14 @@ jobs:
           sh update_version.sh
```

Comment:
the same, change the name of the workflow to point out that it is related to the core module